modules/osdk-run-deployment.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/golang/osdk-golang-tutorial.adoc
+// * operators/operator_sdk/ansible/osdk-ansible-tutorial.adoc
+// * operators/operator_sdk/ansible/osdk-ansible-inside-operator.adoc
+// * operators/operator_sdk/helm/osdk-helm-tutorial.adoc
+
+ifeval::["{context}" == "osdk-golang-tutorial"]
+:golang:
+endif::[]
+
+[id="osdk-run-deployment_{context}"]
+ifeval::["{context}" != "osdk-ansible-inside-operator"]
+= Running as a deployment on the cluster
+endif::[]
+ifeval::["{context}" == "osdk-ansible-inside-operator"]
+= Testing an Ansible-based Operator on the cluster
+
+After you have tested your custom Ansible logic locally inside of an Operator, you can test the Operator inside of a pod on an {product-title} cluster, which is prefered for production use.
+endif::[]
+
+You can run your Operator project as a deployment on your cluster.
+
+ifdef::golang[]
+.Prerequisites
+
+* Prepared your Go-based Operator to run on {product-title} by updating the project to use supported images
```

Comment:
Maybe "You prepared your" 
I'd have to cross-check how other prerequisites are formatted for consistency's sake, so ignore this suggestion if it doesn't align.

Reply:
Looks like you have consistent, parallel usage in other sections. I think my brain just wants these to be complete sentences, but I'm happy as long as we're consistent!